### ISSUE TITLE
Impaired procedural memory

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6772]
**Preferred term label:**
Procedural memory loss

**Synonyms**
Impaired procedural memory

Procedural memory deficit

**Definition (free text, please give PubMed ID)**
A reduction in the ability to retrieve information about how to perform activities, such as how to ride a bike or drive a car, how to perform activities of daily living, or how to play a musical instrument. 

**Parent term (use hpo.jax.org/app)**
Retrograde memory impairment

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
